modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -1484,16 +1484,14 @@ The JSON input payload looks like the following.
 [[skip_on_null]]
 ==== Skip Null On (for JSON)
 
-You can specify whether this generates an outbound message that contains fields
-with "null" values, or if these fields are ignored entirely. This can be set
-through an attribute in the output directive named `skipNullOn`, which can be
-set to three different values: `elements`, `attributes`, or `everywhere`.
+You can use the `skipNullOn` writer property to omit null values from arrays, objects, or both.
 
 When set to:
```

Comment:
Can you tweak this intro to make it a bit more comprehensive? Maybe something like "When blah is set to one of the following values, these actions occur:"? I'm sure you can come up with a better wording than mine!  ;-)